krun/tests/test_scheduler.py

```diff
@@ -237,8 +238,13 @@ def test_pre_and_post_exec_cmds0002(self, monkeypatch, mock_platform):
 
         assert n_reboots == 1
         elems = got.split(":")
-        assert elems[0].endswith(".json.bz2")
-        assert elems[1].endswith(".log")
+        assert os.path.basename(elems[0]) == "one_exec_results.json.bz2"
+        assert re.match("one_exec_[0-9]{8}_[0-9]{6}.log", os.path.basename(elems[1]))
+        assert os.path.basename(elems[2]) == "one_exec.manifest"
+
+        # all paths should be in the same dir
+        dirnames = [os.path.dirname(x) for x in elems]
+        assert all([dirnames[i] == dirnames[0] for i in [1, 2]])
```

Comment:
This `assert` feels weird, particularly with the only-for-the-assert variable above it. Could we not do:

```python
assert os.path.dirname(dirnames[0]) == os.path.dirname(dirnames[1])
assert os.path.dirname(dirnames[0]) == os.path.dirname(dirnames[2])
```